src/map/map_networking.h

```diff
@@ -57,6 +58,8 @@ class MapNetworking
     int32 parse(uint8*, size_t*, MapSession*);                           // main function parsing the packets
     int32 send_parse(uint8*, size_t*, MapSession*, bool);                // main function is building big packet
 
+    int32 send_single_packet_no_pchar(uint8*, size_t*, MapSession*, bool, CBasicPacket*); // used to resend 0x00B if client didn't receive it (dropped packet)
```

Comment:
If you're adding new functions, please use the new naming conventions from newer code like Application.h, etc.
```
    void tapStatistics();
    auto doSocketsBlocking(timer::duration) -> timer::duration;
```

Rather than the old snake_case names. If you're feeling spicy you could even rename all of the outdated function names in here

Reply:
Done, but please review the name again

Reply:
Works for me 👍